### DISCUSSION TITLE
Clarification on Entering IIV for Population Simulation in mrgsolve

### DISCUSSION THREAD

**Discussion by SravaniLanke** [metrumresearchgroup/mrgsolve#1239] [General]
Hi,

I am setting up a population simulation using mrgsolve, and I have a question regarding how to handle inter-individual variability (IIV). Should I enter the IIV reported as CV% directly in the Omega section, or should I convert the CV% to variance (omega^2) before inputting it?

Thank you for your help!

**Comment by kylebaron** [comment]
Hi @SravaniLanke  - 

The OMEGA block takes variances for on-diagonal elements. So if you are entering this information yourself (e.g. pulling values from a publication), then convert %CV to variance before inputting. The off-diagonal elements take covariances or you can optionally enter those as correlations (see the [@correlation](https://mrgsolve.org/user-guide/specification.html#sec-block-omega) block option)

``` r
library(mrgsolve)
#> 
#> Attaching package: 'mrgsolve'
#> The following object is masked from 'package:stats':
#> 
#>     filter
mod <- house()

library(mrgsolve)
mod <- '
$OMEGA @corr
1
0.5 2
'
mod <- mcode("corr", mod)
#> Building corr ...
#> done.
omat(mod)
#> $...
#>          [,1]      [,2]
#> 1:  1.0000000 0.7071068
#> 2:  0.7071068 2.0000000
cov2cor(as.matrix(omat(mod)))
#>      [,1] [,2]
#> [1,]  1.0  0.5
#> [2,]  0.5  1.0
```

<sup>Created on 2024-09-24 with [reprex v2.1.1](https://reprex.tidyverse.org)</sup>

Kyle

**Comment by SravaniLanke** [comment]
Thank You Kyle.
I only have IIV for one parameter, So can I still use the CV% in the correlation matrix. 

Thank you

**Reply by kylebaron** [reply]
If OMEGA is 1x1, then your only OMEGA element is on the diagonal and you'll have to enter that as a variance, converting from CV%.
